app/models/models.py

```diff
@@ -0,0 +1,22 @@
+class Knight:
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line